Libs/MRML/Core/vtkMRMLScene.cxx

```diff
@@ -1350,6 +1350,7 @@ void vtkMRMLScene::RemoveNode(vtkMRMLNode *n)
     this->RemoveReferencesToNode(n);
     }
 
+  n->SetID(NULL);
```

Comment:
It is difficult to ensure that there are no side effects of this.
Would it be possible to fix this issue in volume loading by erasing the node ID after the volume node is removed from the test scene?